### ISSUE TITLE
context menu issue

### ISSUE THREAD

**Issue by ghost** [Yutaka-Sawada/MultiPar#29]
trying to use this in context menu isnnt working for me. I can add it to the copy to context sub menu....but it is not being added to the regular context dropdown menu when right clicking on files. Why is this? I mean for it to be integrated into the shell, but i dont see it in my shell context menu...

**Comment by Yutaka-Sawada** [comment]
> I can add it to the copy to context sub menu....but it is not being added to the regular context dropdown menu when right clicking on files.

I don't know your case's detail. There is an option "Integrate MultiPar into Shell". Did it work ? What do you want to do with the context menu ? Screen-shot and picture may help. 

The context menu depends on Windows Explorer (File Explorer on Windows 10). What OS you use ? If you use 64-bit version OS, you need to use 64-bit shell to see MultiPar's context menu. MultiPar includes Shell Extension DLL for 32-bit and 64-bit. When you use 32-bit shell, you cannot see 64-bit Shell Extension.

**Comment by dabtapp** [comment]
i dont know much about your solution. I can add it to the copy to menu shell. I cannot integrate it. It wont show. It's an 64 bit OS version Version 20H2.  'integrate multipar into shell' is the option i enabled, but it's not showing in the shell.

**Comment by Yutaka-Sawada** [comment]
Because I don't use Windows 10, I don't know that it works on recent OSes. It should work, unless Microsoft changed something in the new OS. When OS is 64-bit, MultiPar must install 64-bit version DLL. If it failed to detect OS, it might install 32-bit version DLL. 

You may try manual installing Shell Extension DLL. At first, un-check the option "Integrate MultiPar into Shell" on MultiPar Option. Then, open "Command Prompt" and change directory to MultiPar's folder. 
```
cd /d C:\somewhere\MultiPar
```

You can register DLL by following command.
```
regsvr32.exe MultiParShlExt64.dll
```

You can remove DLL by following command.
```
regsvr32.exe /u MultiParShlExt64.dll
```

How is the reply of RegSvr32 ? It returns message of success or fail. If it successes, DLL fails something in showing context menu. If it fails, DLL isn't loaded at all.

**Comment by JohnLGalt** [comment]
Running MP **as administrator**, then opening options and enabling "Integrate" into shell works perfectly fine here.

Windows 10 Pro x64 - Ver 20H2 (Build 19042.868).

![MP Shell Ext 005 ](https://user-images.githubusercontent.com/7432512/111814087-6da3b900-88b0-11eb-9f33-30128a6aea17.png)

**Comment by Yutaka-Sawada** [comment]
> Running MP as administrator

Thank you for test on Windows 10. But, I designed that the Shell Extension doesn't require an administrator privilege. It sets keys under HKEY_CURRENT_USER tree. I'm not sure that the system was changed in new OS.

To understand what happened at dabtapp's case, screen-shot will be useful. Such like, how are the context menu. If there are wrong entries in the menu, something error might occur.
